.github/workflows/moving-cards.yml

```diff
@@ -42,20 +43,26 @@ jobs:
       # Now handling the needed labeling
       - name: Triage labeling
         # Only if moved into triage
-        if: ${{ github.event.project_card.column_id == env.TRIAGE_COLUMN_ID  }}
+        if: ${{ github.event.project_card.column_id == env.TRIAGE_COLUMN_ID }}
         uses: andymckay/labeler@1.0.4
         with:
-          add-labels: "triage"
+          add-labels: ${{ needs.get-issue.outputs.creator == 'bitnami-bot' && 'auto-merge' || 'triage' }}
           remove-labels: "on-hold, in-progress, solved"
+      - name: Triage labeling - verify
+        # Only if moved into triage and the creator is bitnami-bot
+        if: ${{ github.event.project_card.column_id == env.TRIAGE_COLUMN_ID && needs.get-issue.outputs.creator == 'bitnami-bot' }}
+        uses: andymckay/labeler@1.0.4
+        with:
+          add-labels: verify
+          # Required to trigger CI workflow
+          repo-token: "${{ secrets.BITNAMI_BOT_TOKEN }}"
```

Comment:
Why do we need to trigger the `labeled` events?

Reply:
`ci-pipeline` is listening for labeled or synchronize events